src/ffi.rs

```diff
@@ -8,10 +8,17 @@ use std::slice;
 use std::fmt;
 use std::os::raw::*;
 use std::ffi::CStr;
-use std::path::*;
 use std::io;
 
 use crate::rustimpl;
+#[cfg(all(target_arch = "wasm32", target_os = "unknown"))]
+mod libc;
+
+#[cfg(all(target_arch = "wasm32", target_os = "unknown"))]
+use std::path::{PathBuf};
```

Comment:
I expect this to break compilation on Windows, since the paths are later used with a different cfg. Also `{Path}` is a redundant syntax.

Reply:
Fixed the `{...}`. Can you clarify the issue with Windows? I'm not very experienced with Rust, let alone conditional compilation, thanks for your patience 😅